_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/styles/rhd-theme/components/_articles.scss

```diff
@@ -1,10 +1,96 @@
-.rhd-p-article {
-  .rhd-c-article__publication-info {
-    font-size: .9em;
-    color: #72767b;
-    font-weight: 500;
-    @extend .pf-u-pb-xl;
+.pf-c-page__sidebar {
```

Comment:
I see this class in 2 scss files, where does it render and why is it hidden by default?